lib/screens/home/pages/stats/stats_page.dart

```diff
@@ -94,7 +95,12 @@ class _StatsPageState extends State<StatsPage> {
             );
           }
           if (state is StatsPageFailure) {
-            return Text(state.message);
+            if (state.message == "The token has expired! Please, login again or refresh it.") {
```

Comment:
We have to think of a better way to do this, comparing the message will cause this feature to break if we decide later to change the message returned by the API. Perhaps now makes sense to return something else added to the message. Can you create an issue on Flutter repo to address this change, please? If not changed now perhaps later, but with an issue, we won't forget about this problem. @Techno-Disaster

Reply:
@anitab-org/mentorship-android-maintainers  any idea how this is done on the android side? Not sure if there is any other alternative right now

Reply:
Not sure, but you can ask on #mentorship-system stream on Zulip. I don't remember now 🤔